cli/commands/bundleCommand/index.js

```diff
@@ -4,13 +4,15 @@ const { OUTPUT_BUNDLE_DIR, ROOT_OUTPUT_DIR, ENTRY_FILE } = require('../../consta
 
 program.command('bundle')
     .description('Creates a CodePush bundle file (assumes Hermes is enabled).')
+    .addOption(new Option('-t, --type <type>', 'project type (react-native | expo)').choices(['react-native', 'expo']).default('react-native'))
```

Comment:
How about changing the name of option to `-f, --framework` and running the expo bundle command only when `-f expo` option passed?

Because I thought `framework` represents Expo’s identity better than `type`.
(I also considered using a `--expo` flag, but it didn’t seem scalable for future frameworks. 😆 )

Reply:
@floydkim 
Awesome! Just switched `type` to `framework` as you suggested.
I've also updated the PR description and the README to reflect these changes.